### PR TITLE
Don't set UICollectionView properties in IGListCollectionView

### LINKED ISSUE #277: Init overrides storyboard settings in IGListCollectionView. Should -[IGListCollectionView commonInit] be removed?
## New issue checklist

- [x] I have reviewed the [`README`](https://github.com/Instagram/IGListKit/blob/master/README.md) and [documentation](http://instagram.github.io/IGListKit)
- [x] I have searched [existing issues](https://github.com/Instagram/IGListKit/issues) and this is not a duplicate

### General information

- `IGListKit` version: 1.0.0
- iOS version(s): 10.1
- CocoaPods/Carthage version:
- Xcode version: 8.1
- Devices/Simulators affected:
- Reproducible in the demo project? (Yes/No): 
- Related issues: #92 

The next release of IGListKit will support storyboards, but at present [-[IGListCollectionView commonInit]](https://github.com/Instagram/IGListKit/blob/f2a9d93be84e7ea23ec85dddbb3e2421215a6eec/Source/IGListCollectionView.m#L15-L16) sets a couple of options which override those provided by the storyboard.

I can open a PR to remove these if appropriate. The IB default appears to be white, but the example ViewControllers would need updating to explicitly set a background color in code.

### PR DESCRIPTION
**Pull request by dcaunt**
Fixes #277

### CODE REVIEW

File: Source/IGListCollectionView.m [hunk 1]
```diff
@@ -11,25 +11,6 @@
...
 @implementation IGListCollectionView
 
-- (void)commonInit {
-    self.backgroundColor = [UIColor whiteColor];
-    self.alwaysBounceVertical = YES;
-}
-
-- (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionViewLayout *)layout {
-    if (self = [super initWithFrame:frame collectionViewLayout:layout]) {
-        [self commonInit];
```
Comment (rnystrom) [inline Source/IGListCollectionView.m]: Can we move all of the `commonInit` into here and just get rid of `initWithCoder:`? I think its nice having a default white bg, saves a lot of boilerplate.

Thoughts @jessesquires?

Comment (dcaunt) [inline Source/IGListCollectionView.m]: FWIW the rationale for removal was that `IGListCollectionView` exists only to provide the compiler warnings but I'm more than happy to update the PR, I agree it's convenient.

Comment (jessesquires) [inline Source/IGListCollectionView.m]: actually, i'm fine either way. 

but i agree that white + bouncing is convenient. 

(also, IG implementations probably rely on these defaults right now?)

Comment (rnystrom) [inline Source/IGListCollectionView.m]: @jessesquires ya and w/out the compiler we will likely break some stuff w/out warning. @dcaunt let's update and maybe open a new issue for that?

I'd love to consider removing the dependence on `IGListCollectionView` but admit I **love** the compiler warnings for those methods.

Comment (jessesquires) [inline Source/IGListCollectionView.m]: agree, we should keep `IGListCollectionView` for those warnings.

but, we should consider #240 if more requests come up.

**Pull request closed without merging**